src/test/java/bowling/domain/ScoreTextTest.java

```diff
@@ -0,0 +1,30 @@
+package bowling.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("ScoreText 테스트")
+class ScoreTextTest {
+
+    @DisplayName("점수를 기호로 변경되는지 확인")
+    @Test
+    void getScoreTextByScore() {
+        for (int i = 0; i < 10; i++) {
+            assertThat(ScoreText.getScoreTextByScore(i)).isEqualTo(ScoreText.values()[i].getScoreText());
+        }
```

Comment:
```suggestion
       assertAll(
                () -> {
                    for (int i = 0; i < 10; i++) {
                        assertThat(ScoreText.getScoreTextByScore(i)).isEqualTo(ScoreText.values()[i].getScoreText());
                    }            
                }
        );
```
assert문이 많아 진다면 위와 같이 assertAll을 사용해보세요.